server/src/task/dto/create-task.request.ts

```diff
@@ -0,0 +1,23 @@
+import { IsMongoId, IsNotEmpty } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class CreateTaskRequest {
+  @ApiProperty({
+    required: true,
+    example: '60c72b2f9b1d8c001c8e4f3a',
+    description: 'ID of the column to which the task is added',
+  })
+  @IsMongoId()
+  @IsNotEmpty()
+  columnId: string;
+
+  //#TODO: Delete when jwt token is implemented
+  @ApiProperty({
+    required: true,
+    example: '60c72b2f9b1d8c001c8e4f32',
+    description: 'ID of the user (manager) who creates the task',
+  })
+  @IsMongoId()
+  @IsNotEmpty()
+  userId: string;
```

Comment:
Both 'columnId' and 'userId' can be converted directly into mongoose.Types.ObjectId but that can be done in the next task.